Gopkg.toml

```diff
@@ -55,42 +30,40 @@
   name = "github.com/pivotal/image-relocation"
   version = "v0.5"
 
-[[constraint]]
-  name = "github.com/deislabs/cnab-go"
-  version = "v0.6.0-beta1"
-
 [[override]]
   name = "github.com/google/go-containerregistry"
   revision = "ef12d49c8daf6a6d72978966d5945e39bb898b4a"
 
-[[override]]
-  name = "k8s.io/apimachinery"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/api"
-  revision = "kubernetes-1.11.2"
-
-[[override]]
-  name = "k8s.io/client-go"
-  revision = "kubernetes-1.11.2"
+[[constraint]]
+  name = "github.com/deislabs/cnab-go"
+  #version = "v0.6.0-beta1"
+  source = "github.com/radu-matei/cnab-go"
+  branch = "deps"
```

Comment:
When there'a new release, we can go ahead and merge this.
See https://github.com/docker/cnab-to-oci/issues/79